### PR TITLE
Issue82: Add Uniswap class

### PR DESCRIPTION
**Pull request by DanilYachmenev**
Issue #82 

Implemented with reference to https://jeiwan.net/posts/programming-defi-uniswap-1/ 

Note that I've created `get_token_amount()` and `get_eth_amount()` instead of `getPrice()` since from what I've read Uniswap is primarily a token exchange platform, so it generally deals in the amount of tokens being exchanged rather than prices - 1 function would suit only for 1 token

### CODE REVIEW

File: defi/uni_swap/uni_swap.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+        :param token_reserve: token reserve
+        :param eth_reserve: ETH reserve
+        """
+        self.token_reserve = token_reserve
+        self.eth_reserve = eth_reserve
+
+    def add_liquidity(self, token_amount: float, eth_amount: float) -> None:
+        """
+        Adds liquidity by increasing the token and ETH reserves.
```
Comment (gpsaggese) [inline defi/uni_swap/uni_swap.py:32]: -> Add

File: defi/uni_swap/uni_swap.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+    def _get_amount(
+        self, input_amount: float, input_reserve: float, output_reserve: float
+    ) -> float:
+        """
+        Get the amount of output token.
+
+        :param input_amount: input token reserve
+        :param input_reserve: output token reserve
+        :param output_reserve: output token reserve
```
Comment (gpsaggese) [inline defi/uni_swap/uni_swap.py:65]: Can you fix / improve this?

**Review by gpsaggese — COMMENTED** [review]
Minor changes

File: defi/uni_swap/uni_swap.py (hunk 2 above)
Comment (DanilYachmenev) [reply defi/uni_swap/uni_swap.py]: extended a bit, do not want to mix with class description
do you want to describe process in more detail?

**Review by tamriq — APPROVED** [review]

**Pull request merged**